include/Megamix/Commands.hpp

```diff
@@ -5,12 +5,18 @@
 
 namespace Megamix{
     enum CustomCommands: u16{
-        VersionNumber = 0x201
+        InputCheck = 0x200,
+        VersionNumber = 0x201,
+        DisplayCondvar = 0x203
```

Comment:
not sure if this should be kept or not - discuss later on (might even need to use a different command range for debug stuff too...?)